omod/src/main/java/org/openmrs/web/controller/user/UserFormController.java

```diff
@@ -142,7 +151,8 @@ public String handleSubmission(WebRequest request, HttpSession httpSession, Mode
 	        @RequestParam(required = false, value = "confirm") String confirm,
 	        @RequestParam(required = false, value = "forcePassword") Boolean forcePassword,
 	        @RequestParam(required = false, value = "roleStrings") String[] roles,
-	        @RequestParam(required = false, value = "createNewPerson") String createNewPerson,
+	        @RequestParam(required = false, value = "createNewPerson") String createNewPerson, 
+	        @RequestParam(required = false, value = "providerCheckBox") String addToProviderTableOption,
```

Comment:
Since it works the way it is, It's really up to you if you want to change, but it's better style to use Boolean for a single checkbox